web/annotation_layer_builder.css

```diff
@@ -271,3 +271,18 @@
   width: 100%;
   height: 100%;
 }
+
+.annotationLayer .annotationTextContent {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  color: transparent;
```

Comment:
I'm assuming that this is necessary for the a11y-engine to actually "find" the elements, and that e.g. `display: none;` doesn't work; is that correct?

Reply:
Yep, this is my understanding of:
https://github.com/mozilla/pdf.js/pull/15237#issuecomment-1200656860

In the future, we could create a specific text layer for the annotation and then make the text selectable, highlightable and then searchable.